content/References/IPv6.md

```diff
@@ -0,0 +1,239 @@
+
+Existing IPv4 network infrastructure components might not be IPv6-compatable.
+This can result in costly changes to upgrade components or replace all or some of enterprise hardware.
+It takes time and effort to train IT professionals on how to implement and manage IPv6 networks.
+
+An IPv6 website or network cannot be seen by or communicated with by an IPv4 network without a gateway.
+IPv4 and IPv6 do not directly communicate with each other without the aid of either a gateway or other network configuration changes like dual-stacking the two protocols within the same network or deploying IPv6 tunnelling to route traffic.
+
+IPv6 requires IP security where IPv4 security is optional. IPv6 uses an IPSec protocol to provide the required IP security.
+For more information, see [IPsec (Internet Protocol Security)](https://www.techtarget.com/searchsecurity/definition/IPsec-Internet-Protocol-Security).
+
+## Converting to IPv6
+Before implementing IPv6, there are a few things to consider:
+
+* Does the network equipment accommodate IPv6?
+  
+  Most modern operating systems (Windows, Linux, Mac) provide IPv6 compatibility, but Internet of Things (IoT) devices, older systems, and legacy network devices that might require attention during a transition from IPv4 to v6 protocol.
+  Organizations need to determine if the return on investment justifies the cost of converting to an all IPv6 infrastructure.  
+  Research your current infrastructure components to determine what is required to implement IPv6, whether it is updating software or firmware or replacing equipment with more modern equivalents.
+
+  Make sure your DNS server is IPv6 capable to provide the name resolution for these more complex addresses.
+  
+  Alternatively, organizations can migrate to IPv6 infrastructures through a three stage approach:
+
+  1. Create a dual-stack network by configuring network components and endpoints to use both IPv6 and IPv4.
+  2. Use an IPv6 tunnel to address the issue of connecting network devices that lack IPv6 support, and allow for gradual replacement of aging IPv4-only network equipment.
+  3. Use NAT Protocol Translation (NAT-PT) as a service to convert IPv4 addresses into IPv6 and vise versa.
+     It operates similarly to how standard NAT translates non-internet routable IPv4 addresses to one or more publicly addressable IP addresses.
+     Setting up NAT-PT gateways can provide smooth transitions for aging hardware to new IPv6-capable alternatives.
+
+* What type of IPv6 address do you want to assign?
+
+  Internal only or publicly routable, for individual nodes or different nodes, broadcast or for point-to-point communication on the same LAN/VLAN.
+
+  There are six types of IPv6 addresses:
+
+  * Global unicast - Addresses that are routable on the Internet and the equivalent of the public IPv4 addresses. These begin with **2001** as the prefix group.
```

Comment:
The global unicast addresses do not always begin with 2001.

Reply:
reworded.